docs/tutorials/uniq-factories/creating-uniq-factories/how-to-create-uniq-metadata.md

```diff
@@ -0,0 +1,438 @@
+    - `Att Name 1`: Optional, the data type of the first attribute of this collection. You can double click on this column to choose from a list of pre-defined data types.
+        - `Att Name 1`: Auto-filled using the data provided in the `Factory` sheet.
+        - `Att Val 1`: Required if the data for `Att Name 1` exists. This column represents the attribute value for the specific attribute defined in `Factory` sheet.
+        - If you have included additional attributes in the `Factory` sheet, you must provide a value for each attribute.
+
+![](./images/metadata-tool-csv-template-tokens-data-added.png)
+![](./images/metadata-tool-csv-template-tokens-attrbs.png)
+
+3.  Once you're done filling out data in the `Tokens` sheet, export the `Tokens` sheet as a CSV file by clicking on `File -> Download -> Comma Separated Values (.csv)`
+4.  Rename the downloaded CSV file to `tokens.csv` and place in your root folder.
+
+<!-- ### Running the tool: -->
+
+## Running the Metadata Tool
+
+Once you have set up your folder structure and created the CSVs for factory and uniqs, the next step is to run the metadata tool. Before we begin, please note the following:
+
+-   A report is always generated during runtime, but it is only saved to a file once you close the tool.
+-   Reports will help you debug your CSV files; any missing information will be noted in the report and sometimes in the terminal/console window.
+
+### Starting the Tool
+
+Starting the tool is different for each operating system. Follow the steps for your operating system:
+
+#### Windows
+
+On Windows you can drag and drop your collection root folder onto the executable. Or, you can open the metadata tool by simply double clicking the executable.
+
+#### Linux
+
+Follow these steps to run the linux binary:
+
+1.  `cd` to the directory where metadata tool binary is present.
+
+2.  Make it executable by running this command in your terminal:
+
+    `sudo chmod +x metadata-tool-v1-4-8-linux`
+
+3.  Start the tool by running this command in your terminal:
+
+    `./metadata-tool-v1-4-8-linux`
+
+#### MacOS
+
+Follow these steps to run the MacOS binary:
+
+1.  `cd` to the directory where metadata tool binary is present.
+
+2.  Make it executable by running this command in your terminal:
+
+    `sudo chmod +x metadata-tool-v1-4-8-macos`
+
+3.  `Ctrl` + `Right Click` on the binary/executable and then click `Open`. It will ask you if you want to open it. Click on `Open` again.
+    ![](./images/metadata-tool-macos-warning.png)
+
+### Working Directory Input
+
+Once you have started the metadata tool, it will prompt you to enter the path to your root folder. Enter the relative, or absolute path to your collection root folder, and press `Enter` key to submit.
+
+![](./images/metadata-tool-path-prompt.png)
```

Comment:
For this image maybe better explain why `../` is used. On Windows if your exe is next to the directory then you can just enter then name like `AngryBananas` and it will work